ecs_client_test.go

```diff
@@ -39,49 +60,205 @@ func TestECSClientRegisterAndDeregisterTaskDefinition(t *testing.T) {
 		HTTPClient: hc,
 	})
 	require.NoError(t, err)
+	require.NotNil(t, c)
 
-	t.Run("RegisterFailsWithInvalidInput", func(t *testing.T) {
-		out, err := c.RegisterTaskDefinition(ctx, &ecs.RegisterTaskDefinitionInput{})
-		assert.Error(t, err)
-		assert.Zero(t, out)
-	})
-	t.Run("DeregisterFailsWithInvalidInput", func(t *testing.T) {
-		out, err := c.DeregisterTaskDefinition(ctx, &ecs.DeregisterTaskDefinitionInput{})
-		assert.Error(t, err)
-		assert.Zero(t, out)
+	registerOut, err := c.RegisterTaskDefinition(ctx, &ecs.RegisterTaskDefinitionInput{
+		ContainerDefinitions: []*ecs.ContainerDefinition{
+			{
+				Command: []*string{aws.String("echo"), aws.String("foo")},
+				Image:   aws.String("ubuntu"),
```

Comment:
Can we use [busybox](https://hub.docker.com/_/busybox) instead of ubuntu? It's a smaller Linux image so ECS can pull the image faster, and we don't need a full ubuntu image to run these tests.